specification/inference/get/GetResponse.ts

```diff
@@ -17,10 +17,11 @@
  * under the License.
  */
 
-import { ModelConfigContainer } from '@inference/_types/Services'
+import { InferenceEndpointContainer } from '@inference/_types/Services'
 
 export class Response {
   body: {
-    models: Array<ModelConfigContainer>
+    /** @aliases models */
```

Comment:
Since we decided to break compatibility, should we keep models as an alias? For what it's worth, the current Python client does the wrong thing and would send `models` as is to the server. That could be fixed in time for 8.15 though.

Reply:
I agree and have removed the alias.

The change breaks compatibility, we can do that because the API is neither GA or heavily used therefore it should be a clean break.